### DISCUSSION TITLE
schema.org included

### DISCUSSION THREAD

**Discussion by ajlowndes** [pretix/pretix#4297] [Q&A]
Looking into implementing pretix. Before I go too far into it, I want to know if events created in pretix will natively include the necessary schema.org data like it says here: https://schema.org/hasCourseInstance#eg-0332. 

Alternatively, is it even necessary? for google seo purposes I mean. It seems like implementing it will improve the searchability of our events a lot, but I don't know if the whole schema.org has fallen out of dev popularity or what.

**Comment by pc-coholic** [comment]
Right now, we do not pass microdata by default - but we are trialing passing https://schema.org/Event

In theory, this should improve listing in Google's search results - but I don't think we've seen much improvement yet - it almost seems like getting properly event-"cards" in the search is a gated feature, that Google needs to manually approve a webpage for...
